vtkext/private/module/vtkF3DRenderPass.cxx

```diff
@@ -211,6 +226,27 @@ void vtkF3DRenderPass::Initialize(const vtkRenderState* s)
     this->MainPass->SetDepthFormat(vtkTextureObject::Fixed32);
   }
 
+  {
+    vtkNew<vtkLightsPass> lightsP;
+    vtkNew<vtkOpaquePass> opaqueP;
+
+    vtkNew<vtkRenderPassCollection> collection;
+    collection->AddItem(lightsP);
+    collection->AddItem(opaqueP);
+
+    vtkNew<vtkSequencePass> sequence;
+    sequence->SetPasses(collection);
+
+    vtkNew<vtkCameraPass> camP;
+    camP->SetDelegatePass(sequence);
+
+    this->MainOnTopPass = vtkSmartPointer<vtkFramebufferPass>::New();
+    this->MainOnTopPass->SetDelegatePass(camP);
+
+    // Needed because VTK can pick the wrong format with certain drivers
+    this->MainOnTopPass->SetDepthFormat(vtkTextureObject::Fixed32);
```

Comment:
is this a VTK issue? Open it ?

Reply:
Ask yourself: https://github.com/f3d-app/f3d/commit/848dde996db85500c601fa30ef112c22d7850380  
I have no idea why it's there tbh.

Reply:
I found some info there: https://gitlab.kitware.com/vtk/vtk/-/merge_requests/10644#note_1437573

But if its not needed lets not add it

Reply:
I don't think you changed it for no reason.  
It makes sense to me to use fixed32 instead of float32.
So, except if you see a good reason to revert to float32, let's keep it as is it.

Reply:
alright